src/Toolbox.php

```diff
@@ -571,10 +571,23 @@ public static function backtrace($log = 'php-errors', $hide = '', array $skip =
     /**
      * Send a deprecated message in log (with backtrace)
      * @param  string $message the message to send
+     * @param  boolean $strict
+     * @param  string $version The version to start the deprecation alert. If null, it is considered deprecated in the current version.
      * @return void
      */
-    public static function deprecated($message = "Called method is deprecated", $strict = true)
+    public static function deprecated($message = "Called method is deprecated", $strict = true, $version = null)
     {
+        $version ??= GLPI_VERSION;
```

Comment:
This should not be necessary. Indeed, the following check is not required if no version is passed to the current method.

```suggestion
```